code/src/containers/forms/new-product-form/NewProductForm.tsx

```diff
@@ -236,7 +244,7 @@ const NewProductForm = () => {
                   helperText={
                     errors.productTranslations &&
                     errors.productTranslations[index]?.description
-                      ? errors.productTranslations[index]?.description.message
+                      ? errors.productTranslations[index].description.message
                       : undefined
```

Comment:
Can we move these variables before `return` statement?

Reply:
Thanks. Yes, we can. I moved it above